### PR TITLE
Apply fixes based on code review

### PR DESCRIPTION
**Pull request by miguelmtzinf**
BGD Labs review: https://governance.aave.com/t/arfc-gho-cross-chain-launch/17616/9

### CODE REVIEW

File: contracts/src/v0.8/ccip/pools/GHO/UpgradeableBurnMintTokenPool.sol [hunk 1]
```diff
@@ -58,9 +59,4 @@ contract UpgradeableBurnMintTokenPool is VersionedInitializable, UpgradeableBurn
   function REVISION() public pure virtual returns (uint256) {
```
Comment (sakulstra) [inline contracts/src/v0.8/ccip/pools/GHO/UpgradeableBurnMintTokenPool.sol]: while this doesn't hurt it also doesn't help and is a bit useless when using `Initializable` imo. I'd rather remove it.

Comment (miguelmtzinf) [reply contracts/src/v0.8/ccip/pools/GHO/UpgradeableBurnMintTokenPool.sol]: removed, ty!

File: contracts/src/v0.8/ccip/pools/GHO/UpgradeableLockReleaseTokenPool.sol [hunk 2]
```diff
@@ -11,21 +13,15 @@ import {IERC20} from "../../../vendor/openzeppelin-solidity/v4.8.3/contracts/tok
...
+/// - Implementation of Initializable to allow upgrades
 /// - Move of allowlist and router definition to initialization stage
 /// - Addition of a bridge limit to regulate the maximum amount of tokens that can be transferred out (burned/locked)
-contract UpgradeableLockReleaseTokenPool is
-  VersionedInitializable,
-  UpgradeableTokenPool,
-  ILiquidityContainer,
-  ITypeAndVersion
-{
+contract UpgradeableLockReleaseTokenPool is Initializable, UpgradeableTokenPool, ILiquidityContainer, ITypeAndVersion {
```
Comment (brotherlymite) [inline contracts/src/v0.8/ccip/pools/GHO/UpgradeableLockReleaseTokenPool.sol:24]: Missing `_disableInitializers()` on the constructor.

File: contracts/src/v0.8/ccip/pools/GHO/UpgradeableBurnMintTokenPool.sol [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
 
 /// @title UpgradeableBurnMintTokenPool
 /// @author Aave Labs
 /// @notice Upgradeable version of Chainlink's CCIP BurnMintTokenPool
 /// @dev Contract adaptations:
-/// - Implementation of VersionedInitializable to allow upgrades
+/// - Implementation of Initializable to allow upgrades
 /// - Move of allowlist and router definition to initialization stage
-contract UpgradeableBurnMintTokenPool is VersionedInitializable, UpgradeableBurnMintTokenPoolAbstract, ITypeAndVersion {
+contract UpgradeableBurnMintTokenPool is Initializable, UpgradeableBurnMintTokenPoolAbstract, ITypeAndVersion {
```
Comment (brotherlymite) [inline contracts/src/v0.8/ccip/pools/GHO/UpgradeableBurnMintTokenPool.sol:20]: Missing `_disableInitializers()` on the constructor.

**Comment by AsafHochstadtCertora** [comment]
After reviewing the changes, the fixes seem ok to me

**Pull request merged**